### DISCUSSION TITLE
PHP容器安装 grpc 扩展后会导致服务器卡死机

### DISCUSSION THREAD

**Discussion by gkShine** [1Panel-dev/1Panel#6363] [Q&A]
### 联系方式

linren_888@126.com

### 1Panel 版本

v1.10.14-lts

### 问题描述

在给 php 容器7 或者 8 添加 grpc 扩展之后，会引起服务器的卡死。

### 重现步骤

在运行环境里的 PHP，创建运行环境扩展，然后扩展内输入 grpc，回车，确定之后，等待制作镜像

**Comment by wanghe-fit2cloud** [comment]
本地环境未复现上述问题。建议先检查一下是否是 1Panel 服务器性能问题导致的（在创建运行环境时，是会根据所选依赖构建 PHP 运行环境镜像）。

**Reply by wanghe-fit2cloud** [reply]
![image](https://github.com/user-attachments/assets/1d9151db-8dfd-4451-afd0-4ac22ab35822)
